src/js/components/Pagination/PaginationStep.js

```diff
@@ -14,12 +14,14 @@ export const PaginationStep = ({
 }) => {
   const { format: formatMessage } = useContext(MessageContext);
   const { theme } = useThemeValue();
+
   return (
     <Box direction="row" align="center" gap="xsmall" {...rest}>
       <Text>{formatMessage({ id: 'pagination.stepLabel', messages })}</Text>
       <Select
         options={options}
         value={step}
+        a11yTitle={formatMessage({ id: 'pagination.stepLabel', messages })}
```

Comment:
looks like some of the files came in from the Pagination PR can we remove and leave those seperate